### ISSUE TITLE
Jobs endpoint 500's for a session with no acquisitions

### ISSUE THREAD

**Issue by coltonlw** [scitran/core#578]
Impact: User interface error popups occur any time a session with no acquisitions is viewed

<img width="1279" alt="screen shot 2016-10-27 at 12 38 48 pm" src="https://cloud.githubusercontent.com/assets/20908947/19778431/596da522-9c42-11e6-8d1e-b9657efd5ff1.png">

TB:

```
2016-10-27 16:37:09      scitran.api                  base.py   288:ERRO Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/webapp2.py", line 570, in dispatch
    return method(*args, **kwargs)
  File "./api/handlers/containerhandler.py", line 242, in get_jobs
    jobs = Queue.search(cont_array, states=states, tags=tags)
  File "./api/jobs/queue.py", line 197, in search
    type1 = containers[0].type
IndexError: list index out of range
 request_id=e45ec17c-1477586229

[pid: 19|app: 0|req: 106/417] 192.168.3.180 () {44 vars in 856 bytes} [Thu Oct 27 16:37:09 2016] GET /api/sessions/5812205ddad42900117f915e/jobs?root=true => generated 58 bytes in 14 msecs (HTTP/1.1 500) 3 headers in 130 bytes (2 switches on core 0)
```

This happens when a request is made to GET /api/sessions/{SessionId}/jobs  for a session which does not have any acquisitions.  

The error occurs when the search method is called with an empty container list (because there are no acquisitions aka containers for that session)
https://github.com/scitran/core/blob/bd0f068cfadbb38cce0ec3566513fd76d7b8ad8b/api/jobs/queue.py#L197

Suggest search method automatically return an empty list if it is called with an empty "containers" argument.

**Comment by coltonlw** [comment]
@ryansanford can you prioritize this, could possibly impact demos?

**Comment by nagem** [comment]
@coltonlw @ryansanford this is fixed in `add-job-info` as part of #552

**Comment by nagem** [comment]
For context, @dpuccetti found this bug as we are allowing session creation in the UI for the first time. Currently there is no way in Flywheel to create a session without an acquisition (although the API supports it). It should not exist as an error in live sites, unless a user is POSTing directly to the API.
